backend/testing/build.gradle

```diff
@@ -1,12 +1,11 @@
 plugins {
-    id 'java-library'
-    id 'kotlin'
+    id "thunderbird.library.jvm"
     alias(libs.plugins.android.lint)
 }
 
 dependencies {
     implementation project(":backend:api")
 
     implementation libs.okio
-    implementation libs.junit
+    implementation libs.bundles.library.jvm.test
```

Comment:
I believe this line can be removed.

Reply:
The dependency is in use, so I will only include the Junit in this case

Reply:
Oh, I see. In this case it's `implementation`, not `testImplementation`.